spec/system/users_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'ログイン', type: :system do
+  it '正常系' do
```

Comment:
```suggestion
  it '正しいパスワードのときはログインできること' do
```
って書きたい

何を検証しているのかわかるように